client/reader/site-stream/featured.jsx

```diff
@@ -94,15 +103,15 @@ class FeedFeatured extends React.PureComponent {
 	}
 }
 
-function mapStateToProps( state, ownProps ) {
-	const { postsStore } = ownProps;
-	const postKeys = postsStore.get();
+function mapStateToProps( state, { streamKey } ) {
+	const stream = getStream( state, streamKey );
+	const postKeys = stream.items;
 	const posts = getPostsByKeys( state, postKeys );
```

Comment:
not really this PR, but are these creating new arrays on every mapping? Should probably be in a memoized selector instead.

Reply:
`getPostsByKeys` is memoized :)